actions/src/main/res/layout/dialog_create_folder.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:background="@drawable/bg_round_white"
```

Comment:
Instead of using this resource you should use `MaterialShapeView` from `base-ui`.
It's a surface that you can assign a shape to and that takes it's properties from the theme.
You can see it used in a couple of places, and it makes sure all non standard shapes have the same roundness value from the theme.

Reply:
Done